app/build.gradle

```diff
@@ -31,33 +32,23 @@ android {
     }
 
     lintOptions {
-        // This is needed to avoid spurious lint errors from libthrift and log4j on android.
+        // This is needed to avoid spurious lint errors from libthrift on android.
         disable 'InvalidPackage'
     }
+
     packagingOptions {
         exclude 'META-INF/LICENSE.txt'
         exclude 'META-INF/NOTICE.txt'
     }
 }
 
-repositories {
-    mavenCentral()
-    maven { url 'https://oss.sonatype.org/content/groups/public' }
-}
-
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-
-    compile 'com.android.support:multidex:1.0.0'
-
-    compile 'com.google.android.gms:play-services-location:6.5.+'
-    compile 'com.google.code.gson:gson:2.2.4'
-
-    compile 'com.amazonaws:aws-android-sdk-core:2.1.3'
+    compile 'com.android.support:multidex:1.0.1'
+    compile 'com.google.code.gson:gson:2.3'
     compile 'com.amazonaws:aws-android-sdk-sqs:2.1.3'
-    compile 'org.slf4j:slf4j-api:1.5.8'
-    compile 'org.slf4j:slf4j-log4j12:1.5.8'
-    compile 'org.apache.thrift:libthrift:0.9.1+'
-    compile 'com.github.kohanyirobert:ebson:0.4-SNAPSHOT'
```

Comment:
We can't remove this dependency. A class not found exception is thrown. But it is thrown in a background thread. So, it does not interfere with the app per se. But this error is critical for the Zendrive SDK and it won't work without this dependency.

java.lang.ClassNotFoundException: com.github.kohanyirobert.ebson.BsonObject
at java.lang.Class.classForName(Native Method)
at java.lang.Class.forName(Class.java:309)
at java.lang.Class.forName(Class.java:273)